pyproject.toml

```diff
@@ -27,30 +27,16 @@ exclude = '''
 [tool.tox]
 legacy_tox_ini = """
 [tox]
-envlist = py35-django22,py3{6,7,8,9,10}-django{22,30,31,32}
+envlist = py3{8,9,10,11,12}-django{40,41,42}
```

Comment:
Are there any customers running apps based on python 3.6 or 3.7 using the django pack that would run into issues deprecating this? We haven't communicated our deprecations well enough to just shut them down IMO.

Reply:
That's fair. We're just removing tests for Python 3.6/3.7 in this case, not actually deprecating support, but i can re-add.

Reply:
To be more clear: these changes don't actually deprecate support for Python 3.6/3.7, they just update our unit tests to drop testing for those environments. Also, Python 3.6/3.7 are officially end-of-life and no longer supported or receiving security updates: https://devguide.python.org/versions/, so it seems safe to drop our testing for those versions?

Reply:
Ah, okay. If they are EOL'ed then that seems safe to drop them.